### PR TITLE
Konan 814 feedback endpoint

### PR DESCRIPTION
**Pull request by mtawfik-95**

### CODE REVIEW

File: konan_sdk/endpoints/base_endpoint.py [hunk 1]
```diff
@@ -96,3 +119,35 @@ def get(self, api_url: str):
         self.process_response()
 
         return self.response
+
+
+class KonanBaseAuthenticatedEndpoint(KonanBaseEndpoint):
+    def __init__(self, api_url: str, user: KonanUser = None, **kwargs) -> None:
+        super().__init__(api_url, user=user, **kwargs)
```
Comment (Kandeel4411) [inline konan_sdk/endpoints/base_endpoint.py:126]: `user=user` might be unnecessary here as `KonanBaseEndpoint` doesn't use it from kwargs or expects it

File: konan_sdk/sdk.py [hunk 2]
```diff
@@ -42,3 +42,14 @@ def predict(self, deployment_uuid: str, input_data: Union[Dict, str]) -> Tuple[s
         prediction_uuid, output = PredictionEndpoint(api_url=self.api_url, user=self.auth.user, deployment_uuid=deployment_uuid).post(payload=input_data)
 
         return prediction_uuid, output
+
+    def feedback(self, deployment_uuid: str, feedback: List[Dict]):
```
Comment (Kandeel4411) [inline konan_sdk/sdk.py:46]: I believe this needs a doc string

Comment (Kandeel4411) [reply konan_sdk/sdk.py:46]: see [this](https://github.com/SynapseAnalytics/konan-sdk/pull/6/files#r777225191)

**Comment by Kandeel4411** [comment]
Other than docs, this LGTM 👍

File: konan_sdk/endpoints/base_endpoint.py (hunk 1 above)
Comment (mtawfik-95) [reply konan_sdk/endpoints/base_endpoint.py:126]: True .. fixed it

File: konan_sdk/sdk.py (hunk 2 above)
Comment (mtawfik-95) [reply konan_sdk/sdk.py:46]: Fixed it

File: konan_sdk/endpoints/base_endpoint.py [hunk 3]
```diff
@@ -1,40 +1,63 @@
...
     Base Endpoint class for Konan endpoints to inherit.
     """
 
-    name = ''  # endpoint name to reference in logs
-
-    def __init__(self, api_url: str, user=None, **kwargs) -> None:
+    def __init__(self, api_url: str, **kwargs) -> None:
+        """Initializes a Konan base endpoint
 
+        Args:
```
Comment (Kandeel4411) [inline konan_sdk/endpoints/base_endpoint.py:18]: I think all the properties require a doc format change as well

**Pull request closed without merging**